pastis/matrix_generation/matrix_from_efields.py

```diff
@@ -367,9 +397,9 @@ def _simulator_matrix_single_mode(which_dm, number_all_modes, wfe_aber, simulato
         hcipy.imshow_field(opd_map, grid=simulator.aperture.grid, mask=simulator.aperture, cmap='RdBu')
         plt.savefig(os.path.join(resDir, 'OTE_images', opd_name + '.pdf'))
 
-    return efield_focal_plane.electric_field
-
+    return efield_focal_plane.electric_field, efield_wfs_plane
 
+#TODO: add norm_one_photon?
```

Comment:
No, this is not needed, as this is a function that becomes a partial function inside the RST matrix class anyway, so it can have a completely different function call. Comment can be deleted.